lib/vmdb/loggers/multicast_logger.rb

```diff
@@ -0,0 +1,31 @@
+class MulticastLogger < Logger
+  attr_accessor :loggers
+
+  def initialize(*loggers)
+    require 'set'
+    @loggers = Set.new(loggers)
+    @level   = DEBUG
+  end
+
+  def level=(new_level)
+    loggers.each { |l| l.level = new_level }
+    super
+  end
+
+  def filename
+    loggers.first.filename
+  end
+
+  [:log_backtrace, :log_hashes].each do |method|
+    define_method(method) do |*args|
+      loggers.map { |l| l.send(method, *args) }.first
```

Comment:
Note, this implies all loggers passed in through `.new` have a log_backtrace and log_hashes method which only our loggers have.  I wonder if we should check if the provided logger implement those methods and only forward the call to them if they implement it.  Maybe that's too much work?  Maybe only support our loggers?  Or just don't care...